### PR TITLE
#150128458 Add get notifications endpoint

### PR DESCRIPTION
**Pull request by Mnickii**
#### What does this PR do?
-Adds get notifications for a logged in user
-Adds a notification endpoint ```/api/v1/notifications```
-Adds a single notification endpoint ```/api/v1/notifications/notification_id```

#### Description of Task to be completed?
-Add get notifications endpoint
-Add get single notification endpoint
-Add notification endpoints tests

#### How should this be manually tested?
At the root of the project directory run the following command:
        
    $ pytest -v 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#150128458

#### Screenshots (if appropriate)
screen shot 2017-09-27 at 3 58 49 pm
screen shot 2017-09-27 at 3 59 15 pm

### CODE REVIEW

**Comment by dennomwas** [comment]
LGTM

File: api/views/contributions.py [hunk 1]
```diff
@@ -140,13 +140,12 @@ def post(self):
             f"New contribution was created by {user_name}",
             "Contribution",
             new_contribution.id,
-            "all",
+            "admin","fellow",
             userId=new_contribution.user_id,
             userName=user_name,
             userImg=""
```
Comment (LarryWachira) [inline api/views/contributions.py]: Why is userImg an empty string?

Comment (Mnickii) [reply api/views/contributions.py]: This was an oversight on my end. I have refactored it to the global user's user image link

File: api/views/notifications.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def get(self, notif_id=None):
+        user_id = g.current_user.user_id
+        if notif_id:
+            notifications = Notification.query_redis(user_id, notif_id)
+            response = {'status': 'sucess', 'data': notifications}
+        else:
+            notifications = Notification.query_redis(user_id)
+            response = {'status': 'sucess',
+                        'data': {'notifications':
+                        [notif for notif in notifications.values()]}}
```
Comment (andela-ojoloko) [inline api/views/notifications.py]: can we move the list comprehension to the models?  We could also replace it with `list(notifications.values())`.

Comment (Mnickii) [reply api/views/notifications.py]: Done @andela-ojoloko

**Comment by andela-ojoloko** [comment]
LGTM

**Pull request merged**